### ISSUE TITLE
Provide vimwiki language escape sequences for characters like $

### ISSUE THREAD

**Issue by chipsenkbeil** [vimwiki/vimwiki#1044]
Per #1025 discussion, should we have escape options for common character sequences to enable using them in a document? Which ones should be done and how would this be handled?

- `\{{{`
- `\}}}`
- `\$`
- `\``
- `\\`
- `\%%`
- `\%%+`
- `\+%%`
- `\{{$`
- `\}}$`
- `\*`
- `\_`
- `\~~`
- `\|`
- `\::`

**Comment by brennen** [comment]
Something like this seems needed and useful.

A single leading backslash as escape for a string of multiple characters feels a bit odd, but I'm not sure what to suggest as an alternative.  A backslash per-character is what I'd expect from other contexts, even though it'd be kind of odd with something like `\{\{\{` where the single `{` by itself isn't significant...

**Comment by chipsenkbeil** [comment]
I picked the backslash because it's a familiar character for use in escape sequences in a variety of languages, but I'm not married to it when it comes to multiple characters. 😄 The only example I have of a multi-character usage with a backslash is unicode such as `\u000A` for line feed.

I did propose a verbatim syntax via #1045 that could be used, but it seems overly verbose to include more than one character to escape.
